From be3630f44ac211a3d88b5bd5974fd14ea5088034 Mon Sep 17 00:00:00 2001
From: Marko Lindqvist <cazfi74@gmail.com>
Date: Sat, 24 Sep 2022 22:53:35 +0300
Subject: [PATCH 28/41] req_copy(): Copy "present"

This was causing barbarian unit build requirement results
to flip occasionally due to use of uninitialized "present" value.

See osdn #45709

Signed-off-by: Marko Lindqvist <cazfi74@gmail.com>
---
 common/requirements.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/common/requirements.c b/common/requirements.c
index 0e1f2dce2f..a9de864423 100644
--- a/common/requirements.c
+++ b/common/requirements.c
@@ -1185,6 +1185,7 @@ void req_copy(struct requirement *dst, const struct requirement *src)
   universal_copy(&(dst->source), &(src->source));
   dst->range = src->range;
   dst->survives = src->survives;
+  dst->present = src->present;
   dst->quiet = src->quiet;
 }
 
-- 
2.35.1

